### PR TITLE
Friday demo

### PR DESCRIPTION
**Pull request by jannelie**
As a developer I want the bot to function in the actual TU-wijk map and not in a test map, so that we can make a nice demo of it.

test should be added later.

### CODE REVIEW

**Comment by nskartoredjo** [comment]
use correct style and use comments

**Comment by nskartoredjo** [comment]
revert `building_plan_demolish(ID) + insert(demolished(ID)).` back.

File: src/main/goal/nl/tudelft/ti2806/serviceKnowledge.pl [hunk 1]
```diff
@@ -36,42 +36,38 @@
...
-	member(indicator(IndicatorId, CurrentValue, TargetValue), IndicatorList).
+	indicator(IndicatorId, CurrentValue, TargetValue, _).
 
 % description:
 % The indicator for our sports center/sportcentrum is stored in here.
 % my_stakeholder_id gives our Stakeholder ID with which we can get our indicator through the indicatorLink.
 % Then the correct indicator is found by matching the IndicatorName.
 sportcentrumIndicator(CurrentValue, TargetValue) :- 
-	my_stakeholder_id(StakeholderId), 
 	indicatorLink(StakeholderId, IndicatorWeights),
```
Comment (HarmenKroon) [inline src/main/goal/nl/tudelft/ti2806/serviceKnowledge.pl]: StakeholderId is not used

File: src/main/goal/nl/tudelft/ti2806/serviceKnowledge.pl [hunk 2]
```diff
@@ -36,42 +36,38 @@
...
-	member(indicator(IndicatorId, CurrentValue, TargetValue), IndicatorList).
+	indicator(IndicatorId, CurrentValue, TargetValue, _).
 
 % description:
 % The indicator for our terrace/terras is stored in here.
 % my_stakeholder_id gives our Stakeholder ID with which we can get our indicator through the indicatorLink.
 % Then the correct indicator is found by matching the IndicatorName.
 terrasIndicator(CurrentValue, TargetValue) :- 
-	my_stakeholder_id(StakeholderId), 
 	indicatorLink(StakeholderId, IndicatorWeights),
```
Comment (HarmenKroon) [inline src/main/goal/nl/tudelft/ti2806/serviceKnowledge.pl]: StakeholderId is not used

**Comment by ardwennem2** [comment]
i approve

File: src/main/goal/nl/tudelft/ti2806/demolish.mod2g [hunk 3]
```diff
@@ -15,5 +15,5 @@ module demolish(ID) {
 	if 
 		bel(buildings(BS), member(building(ID,_,_,_), BS))
 	then 
-		building_plan_demolish(ID).
+		building_plan_demolish(ID) + insert(demolished(ID)).
```
Comment (nskartoredjo) [inline src/main/goal/nl/tudelft/ti2806/demolish.mod2g]: revert back

File: src/main/goal/nl/tudelft/ti2806/serviceKnowledge.pl (hunk 2 above)
Comment (nskartoredjo) [inline src/main/goal/nl/tudelft/ti2806/serviceKnowledge.pl]: efhpoqierugpheriab

File: src/main/goal/nl/tudelft/ti2806/demolish.mod2g (hunk 3 above)
Comment (jannelie) [inline src/main/goal/nl/tudelft/ti2806/demolish.mod2g]: this is now fixed

File: src/main/goal/nl/tudelft/ti2806/serviceKnowledge.pl (hunk 1 above)
Comment (jannelie) [inline src/main/goal/nl/tudelft/ti2806/serviceKnowledge.pl]: this is now fixed

**Comment by jannelie** [comment]
I approve

**Comment by HarmenKroon** [comment]
Looks really nice! I approve

**Comment by nskartoredjo** [comment]
I approve

**Pull request merged**